WalkMe SDK and androidx

We are currently seeing the following error propagate through our build pipeline:
```
/**/node_modules/react-native-walkme-sdk/android/src/main/java/com/walkme/RNWalkMeSdkModule.java:4: error: package android.support.annotation does not exist
import android.support.annotation.Nullable;
                                 ^
/**/node_modules/react-native-walkme-sdk/android/src/main/java/com/walkme/RNWalkMeSdkModule.java:136: error: cannot find symbol
                         @Nullable WritableMap params) {
                          ^
  symbol:   class Nullable
  location: class RNWalkMeSdkModule
Note: /**/node_modules/react-native-walkme-sdk/android/src/main/java/com/walkme/RNWalkMeSdkModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
```

Would it be a heavy lift to port from `android.support.annotation.Nullable` to `android.annotation.Nullable` ?

--- a/android/src/main/java/com/walkme/RNWalkMeSdkModule.java
+++ b/android/src/main/java/com/walkme/RNWalkMeSdkModule.java
@@ -1,7 +1,7 @@
 
 package com.walkme;
 
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
 import android.util.Log;
 
 import com.facebook.react.bridge.ReactApplicationContext;

